bin/rspec

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+#
+# This file was generated by Bundler.
```

Comment:
I'm happy neough to leave them in just in case someone unfamiliar comes across them and is confused or decides to edit them manually.